compiler/src/dmd/cparse.d

```diff
@@ -1914,7 +1914,7 @@ final class CParser(AST) : Parser!AST
                     if (auto p = s.isDeclaration())
                     {
                         auto str = asmName.peekString();
-                        p.mangleOverride = str;
+                        p.mangleOverride = '*' ~ str;
```

Comment:
Don't we have a string concatenation helper that avoids GC calls?

Reply:
I don't think it's necessary here.